codice/ddf
# DDF-4156 Default ingest directory

#### What does this PR do?
1. Causes an empty directory `ingest` to be created off the DDF home directory
2. Changes the `ingest` command so that if no arugment is provided, the default ingest directory is used.

#### Who is reviewing it? 
@austinsteffes @mcalcote @garrettfreibott @ricklarsen @peterhuffer 

#### Ask 2 committers to review/merge the PR and tag them here.
@clockard 
@vinamartin
@tbatie 

#### How should this be tested?
Build distro, unzip and install.
Read the documentation changes.
Verify the DDF_HOME/ingest directory exists.
Put some files in the ingest directory and try and ingest them (use `ingest -t tika`)
Use the ingest command with and without a file or directory.
Try it on both unix and WINDOWS

#### Screenshots:
![screen shot 2018-09-19 at 1 28 43 pm](https://user-images.githubusercontent.com/133284/45779508-10135080-bc10-11e8-8816-2a71a501e5c5.png)

#### Checklist:
- [N/A] Update / Add Unit Tests
- [N/A] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.
